packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -205,6 +205,7 @@ export default async function handleChildrenEventTypes({
               },
             },*/
             hashedLink: hashedLinkQuery(userId),
+            schedulingType: updatedEventType.schedulingType,
```

Comment:
How is this newly added schedulingType property being used ?

Reply:
It isn't used currently, but for database consistency shouldn't this event still have this schedulingType?